core/Cargo.toml

```diff
@@ -35,8 +35,8 @@ void = "1"
 
 [dev-dependencies]
 async-std = { version = "1.6.2", features = ["attributes"] }
-libp2p-mplex = { workspace = true }
-libp2p-noise = { workspace = true }
+libp2p-mplex = { path = "../muxers/mplex" } # Using `path` here because this is a cyclic dev-dependency which otherwise breaks releasing.
+libp2p-noise = { path = "../transports/noise" } # Using `path` here because this is a cyclic dev-dependency which otherwise breaks releasing.
 multihash = { workspace = true , features = ["arb"] }
 quickcheck = { workspace = true }
 libp2p-identity = { workspace = true, features = ["ed25519"] }
```

Comment:
```suggestion
```

Already imported in `[dependencies]`.

Reply:
Will keep as is for now as I don't understand the motivation behind this additional line. See also https://github.com/libp2p/rust-libp2p/pull/4040/files#r1234767608.

Reply:
I don't remember a particular rationale so this is fine to remove.

Reply:
Ah I think I do remember. The `ed25519` feature should not be present in the prod dependencies.